internal/build/image_builder.go

```diff
@@ -354,11 +376,11 @@ func readDockerOutput(ctx context.Context, reader io.Reader, writer io.Writer) (
 		}
 
 		if message.ErrorMessage != "" {
-			return dockerOutput{}, errors.New(message.ErrorMessage)
+			return dockerOutput{}, errors.New(cleanupDockerBuildError(message.ErrorMessage))
 		}
 
 		if message.Error != nil {
-			return dockerOutput{}, errors.New(message.Error.Message)
+			return dockerOutput{}, errors.New(cleanupDockerBuildError(message.Error.Message))
 		}
 
 		if messageIsFromBuildkit(message) {
```

Comment:
i'm confused...so this PR doesn't handle buildkit error messages at all? can you send a snapshot of what this looks like?

Reply:
oh, are these not buildkit errors?

before/after snapshots added to description